### PR TITLE
Add navbarlogo

### PR DESCRIPTION
**Pull request by Dindihub**
Addressing issue #3 

@chalin I'd appreciate your review on this.

### CODE REVIEW

**Comment by chalin** [comment]
> @chalin I'd appreciate your review on this.

Gladly, but the build is failing, so we can't see the preview. I think that it's because you updated Docsy. Maybe wait for the next official Docsy release before updating, or submit the update through a separate PR.

**Comment by chalin** [comment]
For https://app.netlify.com/sites/addtuflogo/deploys/66b28e380699800007468302, try clearing the cache and triggering a rebuild (within Netlify). That might resolve the issue with the Docsy theme repo

File: hugo.yaml
Comment (chalin) [inline hugo.yaml]:
```suggestion
title: TUF
```

File: hugo.yaml
Comment (chalin) [inline hugo.yaml]: Add the project description here.

**Comment by Dindihub** [comment]
> > @chalin I'd appreciate your review on this.
> 
> Gladly, but the build is failing, so we can't see the preview. I think that it's because you updated Docsy. Maybe wait for the next official Docsy release before updating, or submit the update through a separate PR.

Apologies @chalin. I realized too late that it wasn't building. Was to handle it today. Thanks for the suggestions 💯

**Comment by Dindihub** [comment]
@chalin I fixed the logo issue, please review. Thanks

File: assets/scss/_styles_project.scss
Comment (chalin) [inline assets/scss/_styles_project.scss]:
```suggestion
  }

```

File: assets/scss/_styles_project.scss
Comment (chalin) [inline assets/scss/_styles_project.scss]:
```suggestion
        height: 38px;
```

File: hugo.yaml
Comment (chalin) [inline hugo.yaml]:
```suggestion
```

File: hugo.yaml
Comment (chalin) [inline hugo.yaml]:
```suggestion
      description: A Framework for securing software update systems
```

File: assets/scss/_styles_project.scss
Comment (chalin) [inline assets/scss/_styles_project.scss]: I think that this can be removed:

```suggestion
```

File: hugo.yaml
Comment (chalin) [inline hugo.yaml]:
```suggestion
baseURL: https://theupdateframework.io
```

File: assets/icons/tuf-horizontal-white.png
Comment (chalin) [inline assets/icons/tuf-horizontal-white.png:1]: I don't think that this file is being used. If so, let's drop it.

**Review by chalin — APPROVED** [review]
Well done! LGTM after you've addressed the inline comments and suggestions.

File: assets/icons/tuf-horizontal-white.png
Comment (chalin) [reply assets/icons/tuf-horizontal-white.png:1]: Hi. You marked this as resolved but I still see the `.png` file in the PR, and now in the repo. Was there a reason that you kept it?

**Pull request merged**
